scripts/bin/solc-bisect.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
```

Comment:
nit: since `scripts/bin` is added to `$PATH`, I suggest removing `.sh` from the file name, to be able to call `solc-bisect` on the command line, same as `infra`.